ShipManagementService.Infrastructure/Repositories/ShipRepository.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.EntityFrameworkCore;
```

Comment:
Don't you miss a method to obtain ships ?

Reply:
fixed